src/custom_nodes/common/customNodeLibraryInternalManager.cpp

```diff
@@ -21,35 +21,34 @@
 
 namespace ovms {
 namespace custom_nodes_common {
-void CustomNodeLibraryInternalManager::createBuffersQueue(std::string name, size_t singleBufferSize, int streamsLength) {
+void CustomNodeLibraryInternalManager::createBuffersQueue(const std::string name, size_t singleBufferSize, int streamsLength) {
```

Comment:
You have changed `std::string` to `const std::string` which does not change the fact that parameters get copied when passed to functions. Use `const std::string&` to pass by reference and avoid copy overhead.